README.md

```diff
@@ -62,79 +49,49 @@ This should bring source and binary files into subdirectories of `/usr/local/go`
 making it possible to access by adding `github.com/tarantool/go-tarantool` in
 the `import {...}` section at the start of any Go program.
 
-<h2>Hello World</h2>
-
```

Comment:
In the chapter "[Connectors](https://www.tarantool.io/en/doc/latest/getting_started/getting_started_go/)"
of the Tarantool manual you will find an explanation of a very short (18-line)
program written in Go. Follow the instructions at the beginning of the "Connectors"
chapter carefully. Then cut and paste the example into a file named `example.go`,
and run it. You should see: nothing.

If you see that, then you have successfully installed `go-tarantool` and run a 
program that manipulates the contents of a Tarantool database.

Reply:
It is not relevant for proposed changes. These paragraphs were removed.